chaco/tools/tests/image_inspector_test_case.py

```diff
@@ -0,0 +1,213 @@
+""" Tests for the ScatterInspector Chaco tool
```

Comment:
Would you mind renaming this file to `test_image_inspector.py`, so that the standard Python unittest runner will pick it up?